batch/batches/Convert/Engines/KJobConversionEngine.class.php

```diff
@@ -163,10 +163,15 @@ protected function isConversionProgressing($currentModificationTime)
 		if (kFile::checkFileExists($this->inFilePath))
```

Comment:
THis will return true also on existence of a dir, better use checkIsDir also , to verify it is actual file and not a dir